### ISSUE TITLE
Cancelled events have no Start Date and NPE

### ISSUE THREAD

**Issue by alexives** [gdenning/exchange-sync#23]
### Expected behaviour
Does not sync cancelled events

### Actual behaviour
Attempts to sync a cancelled event that somehow has no start date

```
java.lang.NullPointerException: null
        at com.zerodes.exchangesync.calendarsource.google.GoogleCalendarSourceImpl.convertToJodaDateTime(GoogleCalendarSourceImpl.java:322) ~[exchangesync-1.0.6-jar-with-dependencies.jar:na]
        at com.zerodes.exchangesync.calendarsource.google.GoogleCalendarSourceImpl.getAllAppointments(GoogleCalendarSourceImpl.java:145) ~[exchangesync-1.0.6-jar-with-dependencies.jar:na]
        at com.zerodes.exchangesync.sync.SyncCalendarsImpl.generatePairs(SyncCalendarsImpl.java:44) ~[exchangesync-1.0.6-jar-with-dependencies.jar:na]
        at com.zerodes.exchangesync.sync.SyncCalendarsImpl.syncAll(SyncCalendarsImpl.java:98) ~[exchangesync-1.0.6-jar-with-dependencies.jar:na]
        at com.zerodes.exchangesync.App.main(App.java:48) [exchangesync-1.0.6-jar-with-dependencies.jar:na]
```

### Reproduction steps
Unsure how I got a calendar in this state, but I'm bad at calendars. Also I do terrible things to my calendars like running... automated... tools against them... 

Paused at GoogleCalendarSourceImpl.java:146, the event object looks like this:
```
{
   "etag":"2665957396000000",
   "id":"sq9ni1b6cnd5iqritbbm8udst0_20120320T230000Z",
   "kind":"calendar#event",
   "originalStartTime":{
      "dateTime":"2012-03-20T18:00:00.000-05:00"
   },
   "recurringEventId":"sq9ni1b6cnd5iqritbbm8udst0",
   "status":"cancelled"
}
```
I would guess that when it got cancelled, exchange removed the actuall starTime object and changed it to `originalStartTime`. 

✧・ﾟ:\*\\\_(◕ヮ◕)_/\*:・ﾟ✧

Anyway, I've got it fixed, so I'll put up a pull request with the fix.

**Comment by alexives** [comment]
Fixed in #24

**Comment by gdenning** [comment]
Thanks for the bug report and PR Alex!  I decided to fix the NPE in a slightly different way so I'm closing your PR, but thanks again. Let me know if you encounter any other issues.
